src/commands/convert/index.ts

```diff
@@ -137,6 +137,13 @@ export default class Convert extends FingerprintingConvertCommand {
       fs.writeFileSync(checkSuffix(flags.output), JSON.stringify(converter.toHdf()))
       break
     }
+
+    default: {
+      throw new Error(`Unknown filetype provided: ${getInputFilename()}\
```

Comment:
template strings natively support new lines - you don't need to do the backslash + manual newline character paradigm